### DISCUSSION TITLE
uos-server-20-1060a-amd64版本的操作系统无法识别VMWare的通用虚拟网卡

### DISCUSSION THREAD

**Discussion by waitspring** [linuxdeepin/developer-center#7263] [Q&A | 问答板块]
如题：
深度的小伙伴们能帮忙转给统信的同事吗？

**Comment by BLumia** [comment]
UOS 是商业发行版，你直接通过你获取 UOS 的渠道对应的技术支持反馈就可以的呀。

**Reply by waitspring** [reply]
我在UOS的社区提问，被指路到深度的社区来了，深度跟UOS不是一套班子两个产品？

**Reply by BLumia** [reply]
建议联系你购买 UOS 时对应的销售/支持渠道。

UOS 是 deepin 的下游发行版，有很多 deepin 一侧没有的定制或差异，仓库内包的版本也有相当大的差异。这些差异 deepin 这边是不清楚的，因而没法提供有效的支持。

如果你不清楚联系方式的话，官网 <https://www.chinauos.com/> 最下面有一些联系方式可考虑使用。
